examples/cc-template/README.md

```diff
@@ -0,0 +1 @@
+# rules_nixpkgs guide template
```

Comment:
Include a link to the actual guide so that people who discover this README first can find it easily.

Reply:
Done.